qa/suites/rados/singleton-nomsgr/all/pool-access.yaml

```diff
@@ -0,0 +1,9 @@
+roles:
+- [mon.a, osd.0, osd.1, client.0]
+tasks:
+- install:
+- ceph:
```

Comment:
should be 
```yaml
- ceph:
    fs: xfs
```

otherwise we will be using whatever the filesystem `/` is using, instead creating new fs specified here using the available block devices on the test node.

Reply:
Actually, it appears this has historically been worked around with
`- ceph:
    conf:
      global:
        osd max object name len: 460
        osd max object namespace len: 64`